### PR TITLE
Tune Russian phrases a bit

### PR DESCRIPTION
**Pull request by rodionovd**
:ru:

### CODE REVIEW

File: Loading/ru.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -1 +1 @@
-CFBundleName = "Загрузка";CFBundleDisplayName = "Загрузка";NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Программы\nВсе права защищены.";
\ No newline at end of file
+CFBundleName = "Loading";CFBundleDisplayName = "Loading";NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Apps\nВсе права защищены.";
```
Comment (BonzaiThePenguin) [inline Loading/ru.lproj/InfoPlist.strings]: How come these are English? I actually meant to replace "Loading" in Localizable.strings with the translated versions, but forgot.

File: Loading/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -1 +1 @@
...
"SYSTEM" = "Система";

"CHECK_FOR_UPDATES" = "Проверять наличие обновлений";
"SEND_FEEDBACK" = "Оставить отзыв…";
"ABOUT" = "О %@";
"THANKS" = "Спасибо!";

"MORE_INFO" = "Подробнее…";
"DOWNLOAD" = "Загрузить";
"NAME" = "Имя";
```
Comment (BonzaiThePenguin) [inline Loading/ru.lproj/Localizable.strings:1]: Hm, I need to remember to fix the line endings on these files so the diffs will be more readable. The translations look solid, though!

File: Loading/ru.lproj/InfoPlist.strings (hunk 1 above)
Comment (rodionovd) [inline Loading/ru.lproj/InfoPlist.strings]: Ah, so you wanted them to be translated too… I mean, I don't feel that the Russian word for «Loading» would make a good app name, but OK, I'll translate them.

Considering «Bonzai Apps»: I guess it's a company name, so it shouldn't really be translated. Am I wrong?

Comment (BonzaiThePenguin) [inline Loading/ru.lproj/InfoPlist.strings]: You're probably right about the translated product names – I know Apple doesn't translate the name "Finder", although they do translate "Calendar" and "Notes" so I wasn't sure which way I'm supposed to handle it. How does it work if someone doesn't speak English, though? I'm completely clueless about this if you couldn't tell.

For the company name I wasn't 100% sure and had to look it up when I was making the website. I tend to use Bonzai as the trademark part, while Apps would be a generic description. Before this I had Bonzai Games, for example. I dunno.

Comment (rodionovd) [inline Loading/ru.lproj/InfoPlist.strings]: Well, I can assure you that application names in English are totally fine for Russian OS X users. Same goes for company names.

I'll translate the «app» thing right now…

**Comment by BonzaiThePenguin** [comment]
If using English is fine for Russian users then we can go with that!

**Comment by delebedev** [comment]
wow @BonzaiThePenguin @rodionovd, did not notice this PR, please ignore mine.

**Pull request merged**